src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.length > 1 && style.includes(':'));
```

Comment:
It's good that you're using chaining methods, but the indentation of the chained methods could be improved for better readability. Align the methods vertically.